tests/integration/test_vulnerability_detector/conftest.py

```diff
@@ -16,4 +16,7 @@ def restart_modulesd(get_configuration, request):
     truncate_file(LOG_FILE_PATH)
     file_monitor = FileMonitor(LOG_FILE_PATH)
     setattr(request.module, 'wazuh_log_monitor', file_monitor)
-    control_service('start', daemon='wazuh-modulesd')
+    try:
+      control_service('start', daemon='wazuh-modulesd')
+    except:
+        print("Error starting wazuh-modulesd!") 
```

Comment:
Tab sizes must be two. Also, here you're transforming an error in two errors, so I would do this:
```suggestion
    control_service('start', daemon='wazuh-modulesd')
```

Reply:
I am not, actually. Wrapping the error inside that try-except block makes the pytest not to fail if wazuh fails to start. If I remove such a piece of code, a valid test (wazuh can't start because of an invalid configuration) would be marked as failed.

I add this in order ignore the error, maybe you know a better way to do that?

Reply:
Fix tab size: https://github.com/wazuh/wazuh-qa/pull/692/commits/c0089d07c57ccc3b3f50f7aa363aa9ef7bc81218